packages/web/src/components/other/NotFound.jsx

```diff
@@ -13,13 +13,13 @@ const anchorList = [
 
 const NotFound = () => {
     return (
-        <div className="background">
+        <div className="background-not-found">
             <NavBar props={new Props(anchorList, true)} />
             <div className="hubzie-404-image">
                 <img
-                    src="hubzzie.png"
+                    src="/hubzzie.png"
                     className="hubzie-404-image"
-                    alt="Hubzie-404-not-found"
+                    alt="HubzieNotFound"
```

Comment:
Imagine  that you don't have good internet. Alt is going to get rendered instead of the image - what's better `HubzieNotFound` or something like `Hubzie Image - Page not found`